### PR TITLE
Chain Agnostic Standards Alliance Readme

### PR DESCRIPTION
**Pull request by antoineherzog**

### CODE REVIEW

**Comment by antoineherzog** [comment]
Here we go! @ligi @pedrouid

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
...
+- Purpose
+- Structure
+- Meetings
+- Working Groups
+- Members
+- Join
+
+## **Purpose**
+
+The CASA is a autonomous alliance dedicated to write standard for blockchain protocol that are not specific to a single chain. It aims to have collaborators from all the crypto-space. The initiative come from few crypto community members that self-organize with the objective of having unified and efficient process that include the whole ecosystem. It aim to make the crypto more seamless and more interoperable between protocols, software and companies.
```
Comment (ligi) [inline README.md:14]:
```suggestion
The CASA is a autonomous alliance dedicated to write standards for blockchain protocols that are not specific to a single chain. It aims to have collaborators from all the crypto-space. The initiative emerged from a few crypto community members that self-organize with the objective of having unified and efficient process that include the whole ecosystem. It aims to make the crypto more seamless and more interoperable between protocols, software and companies.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,79 @@
...
+
+The CASA have the only goal to organize working group, be a place where all proposals are gathered and to publish final CAIPs and implementation proposals that could be adopted by the community.
```
Comment (ligi) [inline README.md:16]:
```suggestion
The CASA has the only goal to organize working groups, be a place where all proposals are gathered and to publish final CAIPs and implementation proposals that could be adopted by the community. 
```

File: README.md [hunk 3]
```diff
@@ -1 +1,79 @@
...
+The CASA have the only goal to organize working group, be a place where all proposals are gathered and to publish final CAIPs and implementation proposals that could be adopted by the community. 
+
+## **Structure**
+
+The CASA has three levels of participant structure:
+
+| Role                            | Description            |
+| :-------------                  | :-----------              |
+| Members            | All participants. Expected to participate in working groups. |
+| Working Groups            | Collections of members tackling specific areas of interest, such as Ceramic governance or identity. |
```
Comment (ligi) [inline README.md:25]: can you elaborate on "Ceramic"? I am missing context here

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by antoineherzog** [comment]
I fixed all the sentences and removed the sentence about ceramic (it was a remaining copy paste from the template). Please re-check

**Review by ligi — APPROVED** [review]

**Comment by ligi** [comment]
Thanks for the changes - LGTM now - but would love some more eyeballs on it - hope @pedrouid has a chance to review it

**Comment by ligi** [comment]
I will merge end of the week if we get no veto. Last time to chime in @pedrouid @webmaster128 @rekmarks  et all.

**Review by pedrouid — APPROVED** [review]
👍

**Comment by pedrouid** [comment]
LGTM 👍

**Comment by antoineherzog** [comment]
@ligi Can you merge today? I am going to talk about CASA on my speech sunday night at Cosmos Conversation Event :)

**Pull request merged**